change/@azure-msal-node-4e03887c-0f16-4093-9f7d-16dcd91d292e.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

```suggestion
  "type": "none",
```

Just fixing tests, no behavior change in the library itself.